### DISCUSSION TITLE
Necessary if you like using stdout handlers for python logs.

### DISCUSSION THREAD

**Discussion by JiveyGuy** [encode/httpx#3205] [General]
If you want to read things other than httpx logs in your programs then you should use this line:

```python
logging.getLogger("httpx").setLevel(logging.WARNING)
```

**Comment by lovelydinosaur** [comment]
Are we logging more than you'd expect?

**Reply by JiveyGuy** [reply]
So far I have only seen httpx and requests log globally like this. Maybe it is more common in other packages, but most packages that I know of do not produce as much (or any) info-level logging without manual configuration.
